src/processors.zig

```diff
@@ -179,45 +182,50 @@ pub const CrossPageLinkProcessor = struct {
             }
         }
     }
-    fn parseAltText(
-        pctx: anytype,
-        alt_text: []const u8,
-        maybe_scale: ?[]const u8,
-    ) !void {
+    fn parseAltText(pctx: anytype, alt_text: []const u8, maybe_scale: ?[]const u8) !void {
         if (maybe_scale) |scale| {
-            if (parseResolution(scale)) |res| {
-                try pctx.out.print("alt=\"{s}\"", .{alt_text});
-                try pctx.out.print("width=\"{s}\"", .{res[0].?});
-                if (res[1]) |height| try pctx.out.print("height=\"{s}\"", .{height});
-                return;
+            const res = parseResolution(scale);
+            switch (res) {
+                .resolution => {
+                    try pctx.out.print("width=\"{d}\"", .{res.resolution.width});
+                    if (res.resolution.height) |height| try pctx.out.print("height=\"{d}\"", .{height});
```

Comment:
shouldn't `resolution` also return?

Reply:
No because this is scale, which means the second position is probably correct alt text, since we passed the third position over for parsing and we determined its a resolution.
Maybe I should move the 
```zig
            try pctx.out.print("alt=\"{s}\"", .{alt_text});
            return;
```
up for clarity into the resolution block since it'll be equivalent